common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3270,6 +3270,36 @@
     ],
     "sqlState" : "XXKST"
   },
+  "STV2_ENCODER_UNSUPPORTED_PREFIX_KEY" : {
+    "message" : [
+      "Encoder doesn't support prefix key for state store provider: <stateStoreEncoder>. Please use another state store provider that supports prefix keys, or omit the prefix key."
+    ],
+    "sqlState" : "00000"
+  },
+  "STV2_IMPLICIT_KEY_NOT_FOUND" : {
+    "message" : [
+      "Implicit key not found for operation on stateName=<stateName>"
+    ],
+    "sqlState" : "55019"
+  },
+  "STV2_STORE_MULTIPLE_VALUES_PER_KEY" : {
```

Comment:
When using State Store with HDFS, we only support one value per key - we want to throw this error when the user tries to store multiple values per key. This seems to fit 42802 to me